packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -358,6 +376,34 @@ describe('Account', () => {
       }
     }
   })
+
+  it('should listAccountHistory with multiple addresses', async () => {
+    const accountHistoryAll = await client.account.listAccountHistory([addr1, addr2])
+    expect(accountHistoryAll.length).toStrictEqual(34)
+  })
+
+  it('should listAccountHistory with multiple addresses and txtypes', async () => {
+    const accountHistoryAll = await client.account.listAccountHistory([addr1, addr2], { txtypes })
+    expect(accountHistoryAll.length).toStrictEqual(22)
+  })
+
+  it('should listAccountHistory with multiple addresses, txtypes along with index based pagination', async () => {
+    const accountHistoryAll = await client.account.listAccountHistory([addr1, addr2], { txtypes })
+    expect(accountHistoryAll.length).toStrictEqual(22)
+
+    const accountHistory1 = await client.account.listAccountHistory([addr1, addr2], { start: 2, txtypes })
+    expect(accountHistory1[0].txid).toStrictEqual(accountHistoryAll[3].txid)
+    expect(accountHistory1.length).toStrictEqual(19)
+
+    const accountHistory2 = await client.account.listAccountHistory([addr1, addr2], { start: 2, including_start: true, txtypes })
+    expect(accountHistory2[0].txid).toStrictEqual(accountHistoryAll[2].txid)
+    expect(accountHistory2.length).toStrictEqual(20)
+
+    const accountHistory3 = await client.account.listAccountHistory([addr1, addr2], { start: 2, including_start: true, limit: 3, txtypes })
```

Comment:
By default, `including_start` is set to `false`. So that is covered in these tests too, so I didn't add that seperately. 👀 

https://github.com/JellyfishSDK/jellyfish/blob/7433ea47126009577c54b3dd0cba90891c453668/packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts#L391-L396